modules/navier_stokes/test/tests/finite_volume/ins/lid-driven/tests

```diff
@@ -25,20 +32,36 @@
     input = 'lid-driven.i'
     exodiff = 'lid-driven-point-pressure.e'
     requirement = 'The system shall be able to solve the incompressible Navier-Stokes equations in a lid-driven cavity by fixing the point value of the pressure at a certain coordinate.'
-    cli_args = "FVKernels/mean_zero_pressure/type='FVPointValueConstraint' FVKernels/mean_zero_pressure/point='0.001 0.099 0.0' Outputs/file_base='lid-driven-point-pressure'"
+    cli_args = "FVKernels/mean_zero_pressure/type='FVPointValueConstraint' "
+               "FVKernels/mean_zero_pressure/point='0.001 0.099 0.0' "
+               "Outputs/file_base='lid-driven-point-pressure'"
   []
   [point-pressure-action]
     type = 'Exodiff'
     input = 'lid-driven-action.i'
     exodiff = 'lid-driven-point-pressure-action.e'
     requirement = 'The system shall be able to solve the incompressible Navier-Stokes equations in a lid-driven cavity by fixing the point value of the pressure at a certain coordinate using the NSFV action syntax.'
-    cli_args = "Modules/NavierStokesFV/pinned_pressure_type=point-value Modules/NavierStokesFV/pinned_pressure_point='0.001 0.099 0.0' Outputs/file_base='lid-driven-point-pressure-action'"
+    cli_args = "Modules/NavierStokesFV/pinned_pressure_type=point-value "
+               "Modules/NavierStokesFV/pinned_pressure_point='0.001 0.099 0.0' "
+               "Outputs/file_base='lid-driven-point-pressure-action'"
+  []
+  [point-pressure-action-uo]
+    type = 'Exodiff'
+    input = 'lid-driven-action.i'
+    exodiff = 'lid-driven-point-pressure-action.e'
+    requirement = 'The system shall be able to solve the incompressible Navier-Stokes equations in a lid-driven cavity by fixing the point value of the pressure at a certain coordinate using the NSFV action syntax, with a post-treatment of the pressure rather than a Lagrange multiplier-based constraint.'
```

Comment:
```suggestion
    requirement = 'The system shall be able to solve the incompressible Navier-Stokes equations in a lid-driven cavity by fixing the point value of the pressure at a certain coordinate using the Navier-Stokes finite volume action syntax, with a post-treatment of the pressure rather than a Lagrange multiplier-based constraint.'
```